src/FileUpload/PRIVACY.md

```diff
@@ -0,0 +1,24 @@
+# File Upload Privacy
+Disclaimer: This documentation does not warrant completeness or correctness. Please report any missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de) or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+The FileUpload is a special case as it is only processing the upload of files. Therefore the privacy relevant parts take part in other Services and Components like the File System, the Resource Storage or the File Object.
+
```

Comment:
The FileUpload service handles uploading files. 
Services and components which store and use the uploaded files might store, present, delete or export personal data. This is specified in their respective PRIVACY.md.